public/view/overlay-controls/selected-admins-info.jsx

```diff
@@ -22,6 +27,7 @@ var SelectedAdminsInfo = React.createClass({
       <h3>{admin.name}</h3>
       <div>Area: {admin.geo_area_sqkm} km²</div>
       <div>Case data: {this.create_case_data(admin)}</div>
+      <div>Population: {this.commify(admin.population)}</div>
```

Comment:
When this data is missing, I think the frontend will display **Population: NaN**.

Could you change this so that when the data is missing, it either displays something like "Population: *No data.*", or perhaps just doesn't have a "Population" line at all?